blob/blob-client/src/test/java/com/salesforce/multicloudj/blob/client/AbstractBlobStoreIT.java

```diff
@@ -22,13 +22,15 @@
 import com.salesforce.multicloudj.blob.driver.UploadResponse;
 import com.salesforce.multicloudj.common.exceptions.InvalidArgumentException;
 import com.salesforce.multicloudj.common.exceptions.SubstrateSdkException;
+import com.salesforce.multicloudj.common.gcp.GcpConstants;
```

Comment:
this is also not a right dependency, since this a temp workaround, just have a constant local and let's not introduce anti-pattern dependencies